src/main/java/hudson/plugins/jacoco/portlet/JacocoLoadData.java

```diff
@@ -137,7 +131,7 @@ private static void summarize(Map<String, JacocoCoverageResultSummary> summaries
 
     JacocoCoverageResultSummary jacocoCoverageResult = getResult(run);
 	
-	String date = DATE_FORMAT.format(runDate.getTime());
+	String date = new SimpleDateFormat("yyyy-MM-dd").format(runDate.getTime());
```

Comment:
Unfortunately you are mixing many different changes here, not only Java 7->8, these seem to be intended to improve concurrent behavior somehow. This makes review a bit harder, would in general be better to send separate PRs for these unrelated changes.

Reply:
This is not unrelated. Yet again caused by the switch to org.jenkins-ci.plugins 3.0 and the underlying FindBugs analysis. If not fixed, the build fails.